workspaces/azureml_devtestlabs/install_workspace_vanilla.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+porter install tre-workspace-vanilla --reference "${ACR_NAME}.azurecr.io/tre-workspace-vanilla:v0.1.0" \
```

Comment:
consider making the code DRY as the file above is almost the same and exactly the same if the parameters are abstracted as inputs